src/Core/MainLoop.js

```diff
@@ -92,8 +92,26 @@ MainLoop.prototype._step = function _step(view, timestamp) {
 
     // update camera
     const dim = this.gfxEngine.getWindowSize();
+    const foo = (dim.width != view.camera.width || dim.height != view.camera.height);
```

Comment:
Let's find a better name than `foo`. `foo` is Sacred™, let's use it only to praise the Gods of Coding ;-)

Reply:
oops... sorry about that, didn't realize it was still here !